catalog/imaging/imaging-actionprovider-chip/src/main/java/org/codice/alliance/imaging/chip/actionprovider/ImagingChipActionProvider.java

```diff
@@ -145,71 +148,86 @@ public String getId() {
 
   @Override
   public <T> boolean canHandle(T subject) {
-    boolean canHandle = false;
-
     if (subject instanceof Metacard) {
       Metacard metacard = (Metacard) subject;
 
       boolean isImageNitf = NITF_IMAGE_METACARD_TYPE.equals(metacard.getMetacardType().getName());
       boolean hasLocation = hasValidLocation(metacard.getLocation());
-      boolean hasDerivedImage = hasOriginalDerivedResource(metacard);
-
-      canHandle = isImageNitf && hasLocation && hasDerivedImage;
+      // The chipping transformer requires the NITF resource, the overview derived image
+      // resource, and the original derived image resource.
+      boolean hasNitfResource = metacard.getResourceURI() != null;
+      boolean hasOverviewDerivedImageResource =
+          getDerivedResourceUriByQualifier(metacard, Metacard.DERIVED_RESOURCE_URI, "original")
```

Comment:
Think there might be constants defined somewhere for `original` and `overview`. Not a big deal.

Reply:
It doesn't look like there are good constants to grab. We usually just redefine the "overview" and "original" constants in Alliance.